tests/index.html

```diff
@@ -20,8 +20,8 @@
 		mocha.run();
 	});
 	</script>
-	<script src="../cookie.js"></script>
-	<script src="spec.js"></script>
+	<script src="../dist/cookie.umd.js"></script>
+	<script src="./spec.js"></script>
```

Comment:
Does the `./` make any difference? It looks like it should work without.

Reply:
Thanks! Updated: https://github.com/florian/cookie.js/pull/47/commits/1447fc5f1b16e2fb50866850b40e0175de3c6f22

Reply:
Thanks!